src/ui.rs

```diff
@@ -300,7 +301,7 @@ impl Button {
             pos,
             text: text.into(),
             color: palette.gui_text,
-            ..Default::default()
+            ..Button::default()
```

Comment:
I prefer having `Default::default` here again. This is just repeating the type for no clear good reason.